tidy3d/components/dispersion_fitter.py

```diff
@@ -363,7 +363,7 @@ def get_default_weights(cls, eps: ArrayComplex1D) -> Tuple[float, float]:
     def pole_residue(self) -> Tuple[float, ArrayComplex1D, ArrayComplex1D]:
         """Parameters for pole-residue model in original units."""
         if self.eps_inf is None or self.poles is None:
-            return None
+            return PoleResidue(eps_inf=1, poles=[])
```

Comment:
`PoleResidue` is not defined, but when imported at the module level causes a circular dependency. Could you double-check @caseyflex?

Reply:
should be ok now, thanks for your work on the conflicts!

Reply:
awesome, thanks! this is probably ready to go then @momchil-flex